client/components/language-picker/modal.jsx

```diff
@@ -417,7 +416,17 @@ export class LanguagePickerModal extends PureComponent {
 		this.handleClose();
 	};
 
-	handleClose = () => {
+	recordLanguagePickerEvent = ( isClosingWithoutSelection ) => {
+		this.props.recordTracksEvent( 'calypso_languagepicker_closed', {
+			searched: false !== this.state.search,
+			is_closing_without_selection: isClosingWithoutSelection,
+			initially_selected_language: this.initiallySelectedLanguageSlug,
```

Comment:
Do you know if there's any idea behind keeping the initially selected language and the current selected language in separate props? 

I'm asking because I can't think of a use for those in the potential analysis of this event, and I'm wondering whether a flag field like `selected_language` that looks at whether `initiallySelectedLanguageSlug` is different from `selectedLanguageSlug` could be more useful.

Reply:
Sure, `selected_language` would presumably mean "did the user select a langauge" and would be a boolean?